mono/eglib/glib.h

```diff
@@ -1133,6 +1139,30 @@ gboolean   g_file_test (const gchar *filename, GFileTest test);
 
 gchar *g_mkdtemp (gchar *tmpl);
 
+static inline int
+g_async_safe_fgets (char *str, int num, int handle)
+{
+	int without_padding = num - 1;
+	int i=0;
+	char scratch [2];
+	while (i < without_padding && read (handle, scratch, 1)) {
```

Comment:
`s/read/g_read`?

Reply:
Ah good catch

Reply:
Really? Read is standard Unix. Just that it is _read on Windows, which isn't relevant.

Reply:
@jaykrell I think it's more of a greppability and consistency thing

Reply:
I didn't think of that. I thought that is what the spaces are for.